<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>My_Ui</class>
 <widget class="QDialog" name="My_Ui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>660</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>My_Ui</string>
  </property>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>140</width>
     <height>160</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <item>
    <property name="text">
     <string>基本信息</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>学习情况</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>爱好特长</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>联系方式</string>
    </property>
   </item>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>10</y>
     <width>491</width>
     <height>351</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="lineWidth">
    <number>20</number>
   </property>
   <property name="midLineWidth">
    <number>27</number>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QLabel" name="label_name">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>60</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>姓名</string>
     </property>
     <property name="buddy">
      <cstring>lineEdit_name</cstring>
     </property>
    </widget>
    <widget class="QLabel" name="label_number">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>60</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>学号</string>
     </property>
     <property name="buddy">
      <cstring>lineEdit_number</cstring>
     </property>
    </widget>
    <widget class="QLabel" name="label_gender">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>60</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>性别</string>
     </property>
     <property name="buddy">
      <cstring>comboBox_gender</cstring>
     </property>
    </widget>
    <widget class="QLabel" name="label_birthday">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>60</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>生日</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_hp">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>210</y>
       <width>60</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>头像</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_name">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>50</y>
       <width>150</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_number">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>110</y>
       <width>150</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_gender">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>170</y>
       <width>60</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <item>
      <property name="text">
       <string>男</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>女</string>
      </property>
     </item>
    </widget>
    <widget class="QDateEdit" name="dateEdit_birthday">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>220</y>
       <width>150</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_photo">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>220</y>
       <width>60</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>…</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_picture">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>30</y>
       <width>191</width>
       <height>181</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::WinPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="lineWidth">
      <number>40</number>
     </property>
     <property name="midLineWidth">
      <number>40</number>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>110</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>专业</string>
     </property>
     <property name="buddy">
      <cstring>lineEdit_major</cstring>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>110</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>入学时间</string>
     </property>
     <property name="buddy">
      <cstring>lineEdit_time</cstring>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>210</y>
       <width>110</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>学制</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>160</y>
       <width>110</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>学历</string>
     </property>
     <property name="buddy">
      <cstring>radioButton_3</cstring>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_major">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>40</y>
       <width>150</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_time">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>110</y>
       <width>150</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="inputMask">
      <string>9999年99月99日</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>150</y>
       <width>221</width>
       <height>61</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QRadioButton" name="radioButton_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>150</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>专科</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_4">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>20</y>
        <width>150</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>本科</string>
      </property>
     </widget>
    </widget>
    <widget class="QSpinBox" name="spinBox_year">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>220</y>
       <width>50</width>
       <height>30</height>
      </rect>
     </property>
     <property name="wrapping">
      <bool>false</bool>
     </property>
     <property name="frame">
      <bool>true</bool>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::UpDownArrows</enum>
     </property>
     <property name="accelerated">
      <bool>false</bool>
     </property>
     <property name="correctionMode">
      <enum>QAbstractSpinBox::CorrectToPreviousValue</enum>
     </property>
     <property name="keyboardTracking">
      <bool>true</bool>
     </property>
     <property name="suffix">
      <string>年</string>
     </property>
     <property name="minimum">
      <number>2</number>
     </property>
     <property name="maximum">
      <number>5</number>
     </property>
     <property name="stepType">
      <enum>QAbstractSpinBox::DefaultStepType</enum>
     </property>
     <property name="value">
      <number>2</number>
     </property>
    </widget>
    <widget class="QSlider" name="Slider">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>260</y>
       <width>160</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>2</number>
     </property>
     <property name="maximum">
      <number>5</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_3">
    <widget class="QCheckBox" name="checkBox_all">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>260</y>
       <width>150</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>全选</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>30</y>
       <width>401</width>
       <height>211</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <widget class="QCheckBox" name="checkBox_ball">
      <property name="geometry">
       <rect>
        <x>231</x>
        <y>113</y>
        <width>77</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>足球</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_cartoon">
      <property name="geometry">
       <rect>
        <x>231</x>
        <y>67</y>
        <width>77</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>动漫</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_photo">
      <property name="geometry">
       <rect>
        <x>231</x>
        <y>159</y>
        <width>77</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>摄影</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_painting">
      <property name="geometry">
       <rect>
        <x>231</x>
        <y>21</y>
        <width>77</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>绘画</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_game">
      <property name="geometry">
       <rect>
        <x>21</x>
        <y>113</y>
        <width>127</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>网络游戏</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_film">
      <property name="geometry">
       <rect>
        <x>21</x>
        <y>159</y>
        <width>102</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>看电影</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_travel">
      <property name="geometry">
       <rect>
        <x>21</x>
        <y>21</y>
        <width>77</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>旅游</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_book">
      <property name="geometry">
       <rect>
        <x>21</x>
        <y>67</y>
        <width>77</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>看书</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="page_4">
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>60</y>
       <width>150</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>电子邮箱</string>
     </property>
     <property name="buddy">
      <cstring>lineEdit_email</cstring>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>130</y>
       <width>150</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>联系电话</string>
     </property>
     <property name="buddy">
      <cstring>lineEdit_phone</cstring>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_email">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>60</y>
       <width>150</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_phone">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>130</y>
       <width>150</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>listWidget</sender>
   <signal>currentRowChanged(int)</signal>
   <receiver>stackedWidget</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>125</x>
     <y>151</y>
    </hint>
    <hint type="destinationlabel">
     <x>188</x>
     <y>243</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Slider</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>spinBox_year</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>341</x>
     <y>277</y>
    </hint>
    <hint type="destinationlabel">
     <x>327</x>
     <y>246</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
